lib/pdf417.js

```diff
@@ -4059,8 +4059,8 @@ var PDF417 = {
       return false;
     }
     // get the input sequence array
-    sequence = this.getInputSequences(code);
-    codewords = []; // array of code-words
+    const sequence = this.getInputSequences(code);
+    let codewords = []; // array of code-words
```

Comment:
Follow the original changes of the pdf417-generator
Ref: https://github.com/pkoretic/pdf417-generator/blob/master/lib/pdf417.js#L443-L444